### DISCUSSION TITLE
I lost admin user? :: 4.0.0.rc3

### DISCUSSION THREAD

**Discussion by iferca** [mastodon/mastodon#20585] [Q&A] [answered]
I cannot find ways to change server rules, banners and instance descriptions in the new UI. Is it a documentation issue or a bug? 
I've created a new instance and since I'm learning my way around the administration of the service I decided to start with the new 4.0 branch. As stated I'm running the (at the moment) latest RC3 tag. 
I logged in as the admin user and I can only find account related configuration, nor instance or modetation settings. At some point I thought my user was not actually an admin so I run mastodos:setup again (since I've hit[ this issue](https://github.com/mastodon/mastodon/issues/18813) with no solution following the given recommendations), same results with the new admin user.

**Comment by iferca** [comment]
Actually... I'm back to my theory that the issue is related to the privs of the "admin" user. I tried accessing /admin/ URL (just trying random things) and I see this error in the logs: 

`path=/admin/dashboard format=html controller=Admin::DashboardController action=index status=403 duration=24.71 view=2.18 db=4.38`

how can I make sure a user is admin? 

```
mastodon@localhost:~/live$ tootctl accounts modify root --role admin
Cannot find user role with that name
mastodon@localhost:~/live$ tootctl accounts modify root --role Admin
Cannot find user role with that name
```

**Reply by Tslmuun** [reply]
you need owenr account .

tootctl accounts modify username --role owenr

**Reply by YJSoft** [reply]
It's `Owner` not `owenr`

**Reply by Tslmuun** [reply]
> 这`Owner`不是`owenr`

oh……………


Sorry.

**Reply by iferca** [reply]
Thanks! solved!

**Comment by fwenzel** [comment]
I have the same problem - after upgrading to v.4.0.0rc3 I lost my admin privileges. I can use my own instance, but I cannot administer it.

```
docker-compose run --rm web bin/tootctl accounts modify fred --role Admin
Cannot find user role with that name

docker-compose run --rm web bin/tootctl accounts modify fred --role Owner
Cannot find user role with that name
```

Any hints? Can I / do I need to manually create the default roles?

**Reply by fwenzel** [reply]
Update to my own post -- apparently a random sprinkling of db migrations had not actually been run (???) despite me running db:migrate

```
[...]
  down    20220527114923  Remove filtered languages from users
   up     20220606044941  Create webhooks
   up     20220611210335  Create user roles
   up     20220611212541  Add role id to users
   up     20220613110628  Create custom filter keywords
   up     20220613110711  Migrate custom filters
  down    20220613110802  Remove whole word from custom filters
   up     20220613110834  Add action to custom filters
  down    20220613110903  Remove irreversible from custom filters
  down    20220617202502  Migrate roles
  down    20220704024901  Migrate settings to user roles
   up     20220710102457  Add display name to tags
   up     20220714171049  Create tag follows
  down    20220729171123  Fix custom filter keywords id seq
   up     20220808101323  Create custom filter statuses
   up     20220824164433  Add human identifier to admin action logs
  down    20220824164532  Remove recorded changes from admin action logs
   up     20220824233535  Create status trends
   up     20220827195229  Change canonical email blocks nullable
   up     20220829192633  Add languages to follows
   up     20220829192658  Add languages to follow requests
   up     20221006061337  Create preview card trends
   up     20221012181003  Add blurhash to site uploads
   up     20221021055441  Add index featured tags on account id and tag
   up     20221025171544  Add index ip blocks on ip
  down    20221101190723  Backfill admin action logs
   up     20221104133904  Add name to featured tags
```

Running db:migrate again seems to have caught up. Weird, but I hope this helps people who search for this issue.

**Reply by chetwisniewski** [reply]
This resolved my issue. Thanks

**Reply by joncamfield** [reply]
Specifically, running db_migrate WITHOUT the SKIP_POST_DEPLOYMENT_MIGRATIONS=true var, e.g.:

mastodon$ RAILS_ENV=production bundle exec rails db:migrate

**Comment by iferca** [comment, marked as answer]
[This](https://github.com/mastodon/mastodon/discussions/20585#discussioncomment-4130502) solved the issue
